src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -5964,4 +5964,25 @@ Stack<T> GetNumbers<T>(T[] values)
             ReferenceAssemblies = ReferenceAssemblies.Net.Net80,
         }.RunAsync();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/77962")]
+    public async Task TestNotOnBindingList()
+    {
+        await new VerifyCS.Test
+        {
+            TestCode = """
+                using System.ComponentModel;
+                
+                class C
+                {
+                    BindingList<T> GetNumbers<T>(T[] values)
+                    {
+                        return new BindingList<T>(values);
+                    }
+                }
+                """,
+            LanguageVersion = LanguageVersion.CSharp12,
+            ReferenceAssemblies = ReferenceAssemblies.Net.Net80,
+        }.RunAsync();
```

Comment:
BindingList already derives from Collection, which we already check for with:

```c#
            var collectionType = compilation.CollectionOfTType();
            var blockingCollectionType = compilation.BlockingCollectionOfTType();
            if (constructedType.GetBaseTypesAndThis().Any(
                    t => Equals(collectionType, t.OriginalDefinition) ||
                         Equals(blockingCollectionType, t.OriginalDefinition)))
            {
                return false;
            }
```

so we don't need to do anything else.